specification/search/Azure.Search/models-index.tsp

```diff
@@ -0,0 +1,1077 @@
+   * organized by each applicable field; null if hit highlighting was not enabled
+   * for the query.
+   */
+  @visibility("read")
+  @encodedName("application/json", "@search.highlights")
+  highlights?: Record<string[]>;
+
+  /**
+   * Captions are the most representative passages from the document relatively to
+   * the search query. They are often used as document summary. Captions are only
+   * returned for queries of type 'semantic'.
+   */
+  @visibility("read")
+  @encodedName("application/json", "@search.captions")
+  captions?: QueryCaptionResult[];
+}
+
+/**
+ * Captions are the most representative passages from the document relatively to
+ * the search query. They are often used as document summary. Captions are only
+ * returned for queries of type `semantic`.
+ */
+#suppress "@azure-tools/typespec-azure-core/bad-record-type" "model should have additionalProperties"
+model QueryCaptionResult {
+  ...Record<unknown>;
+
+  /**
+   * A representative text passage extracted from the document most relevant to the
+   * search query.
+   */
+  @visibility("read")
+  text?: string;
+
+  /**
+   * Same text passage as in the Text property with highlighted phrases most
+   * relevant to the query.
+   */
+  @visibility("read")
+  highlights?: string;
+}
+
+/**
+ * Response containing suggestion query results from an index.
+ */
+model SuggestDocumentsResult {
+  /**
+   * The sequence of results returned by the query.
+   */
+  @visibility("read")
+  @encodedName("application/json", "value")
+  results: SuggestResult[];
+
+  /**
+   * A value indicating the percentage of the index that was included in the query,
+   * or null if minimumCoverage was not set in the request.
+   */
+  @visibility("read")
+  @encodedName("application/json", "@search.coverage")
+  coverage?: float64;
+}
+
+/**
+ * A result containing a document found by a suggestion query, plus associated
+ * metadata.
+ */
+#suppress "@azure-tools/typespec-azure-core/bad-record-type" "model should have additionalProperties"
+model SuggestResult {
+  ...Record<unknown>;
+
+  /**
+   * The text of the suggestion result.
+   */
+  @visibility("read")
+  @encodedName("application/json", "@search.text")
+  text: string;
+}
+
+/**
+ * Parameters for filtering, sorting, fuzzy matching, and other suggestions query
+ * behaviors.
+ */
+model SuggestRequest {
+  /**
+   * An OData expression that filters the documents considered for suggestions.
+   */
+  filter?: string;
+
+  /**
+   * A value indicating whether to use fuzzy matching for the suggestion query.
+   * Default is false. When set to true, the query will find suggestions even if
+   * there's a substituted or missing character in the search text. While this
+   * provides a better experience in some scenarios, it comes at a performance cost
+   * as fuzzy suggestion searches are slower and consume more resources.
+   */
+  @encodedName("application/json", "fuzzy")
+  useFuzzyMatching?: boolean;
```

Comment:
Same here why are we doing something different than what's send over the wire?

Reply:
If this is for client libraries then you can just leave this as fuzzy and add `@@clientName(<model>.fuzzy, "useFuzzyMatching")` and it will apply cross-language unless you scope it down to a specific language